Src/BootCamp.Chapter/Inventory.cs

```diff
@@ -2,25 +2,64 @@
 {
     public class Inventory
     {
+        private const int defaultInventorySize = 10;
```

Comment:
I know it wasn't part of the requirements that an inventory should have a size but I've wanted to include that extra bit in the code so I can get a better understanding of how it should be done in a real game with certain amount of slots in an inventory.

It also makes adding/removing items a bit easier as I don't have to cache a copy of the inventory and then recreate a new inventory with a size that accounts for that new/deleted item. Expanding the inventory with upgrades, however, will definitely need this, though it wasn't part of this hw.